packages/e2e/features/ui/components/authenticator/reset-password.feature

```diff
@@ -7,16 +7,16 @@ Feature: Reset Password
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-totp-mfa"
 
-  @next @react @vue @angular
```

Comment:
We can remove `@react` since we use `@next` to run our e2e tests against our Next examples.

Reply:
As mentioned above, the docs' framework drop-down & content relies on these tags to know that `@react` and `@next` frameworks both support this feature:

https://github.com/aws-amplify/amplify-ui/blob/911c82c2c7e2985bd0b8a6d4cbbe92a2fde7b53d/docs/src/components/Feature.tsx#L76-L84

e2e tests aren't running any `react` examples:

https://github.com/aws-amplify/amplify-ui/blob/911c82c2c7e2985bd0b8a6d4cbbe92a2fde7b53d/.github/workflows/tests.yml#L55-L60

Reply:
We can just update the docs to point to `@next` by default. I'm not sure I understand when we would want docs to show React supports a feature that Next doesn't?